### DISCUSSION TITLE
Error 255 during Script cache:clear

### DISCUSSION THREAD

**Discussion by pascal541** [kimai/kimai#4102] [On-Premise: Installation & Updates]
I am trying to update Kimai from 1.30.11 to 2.0.x. After running the `composer install --optimize-autoloader -n` command it runs `cache:clear` which is giving a error (see screenshot). I am on PHP 8.1. Any help is appreciated.
![Schermafbeelding 2023-05-28 212816](https://github.com/kimai/kimai/assets/7486865/20b12845-625c-4347-b2c9-e8c92beb3222)

**Comment by kevinpapst** [comment]
https://www.kimai.org/en/download.html
![Bildschirmfoto 2023-06-12 um 11 47 26](https://github.com/kimai/kimai/assets/533162/7c5913bb-efb1-4f1a-b450-05f49f7e72aa)

https://github.com/kimai/kimai#requirements
![Bildschirmfoto 2023-06-12 um 11 47 39](https://github.com/kimai/kimai/assets/533162/30e85dcc-ec7d-46a4-8aa1-ce0503a08ceb)

**Comment by pascal541** [comment]
Sorry, I'm on PHP 8.1.

**Reply by kevinpapst** [reply]
Then try `rm -r var/cache/*` and run the composer command again

**Reply by pascal541** [reply]
Just tried that, still the same problem

**Reply by kevinpapst** [reply]
Show me `php -m`

**Reply by pascal541** [reply]
```
[PHP Modules]
bcmath
bz2
calendar
Core
ctype
curl
date
dba
dom
enchant
exif
fileinfo
filter
ftp
gd
gettext
hash
iconv
imap
intl
json
ldap
libxml
mbstring
mysqli
mysqlnd
odbc
openssl
pcntl
pcre
PDO
pdo_mysql
PDO_ODBC
pdo_pgsql
pdo_sqlite
pgsql
Phar
posix
pspell
readline
Reflection
session
shmop
SimpleXML
snmp
soap
sockets
SPL
sqlite3
standard
sysvmsg
sysvsem
sysvshm
tidy
tokenizer
xml
xmlreader
xmlwriter
xsl
zip
zlib
```

**Reply by kevinpapst** [reply]
Nothing fishy as well 🤷
